docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/AveragedPerceptron.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.ML;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class AveragedPerceptron
+    {
+        public static void Example()
+        {
+            // In this examples we will use the adult income dataset. The goal is to predict
+            // if a person's income is above $50K or not, based on different pieces of information about that person.
+            // For more details about this dataset, please see https://archive.ics.uci.edu/ml/datasets/adult
+
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Download and featurize the dataset
+            var data = SamplesUtils.DatasetUtils.LoadFeaturizedAdultDataset(mlContext);
+
+            // Leave out 10% of data for testing
+            var trainTestData = mlContext.BinaryClassification.TrainTestSplit(data, testFraction: 0.1);
+
+            // Create data training pipeline
+            var pipeline = mlContext.BinaryClassification.Trainers.AveragedPerceptron(
+                                        "IsOver50K", "Features", numIterations: 10);
+
+            // Fit this pipeline to the training data
+            var model = pipeline.Fit(trainTestData.TrainSet);
+
+            // Evaluate how the model is doing on the test data
+            var dataWithPredictions = model.Transform(trainTestData.TestSet);
+            var metrics = mlContext.BinaryClassification.EvaluateNonCalibrated(dataWithPredictions, "IsOver50K");
+            SamplesUtils.ConsoleUtils.PrintMetrics(metrics);
+
+            // Output:
```

Comment:
>Output [](start = 15, length = 6)

Expected output, then an extra space on the output below.

Reply:
done

---
In reply to: [256221990](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256221990) [](ancestors = 256221990)

Reply:
Add space between `Expected output` and line below it. 

---
In reply to: [256538945](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256538945) [](ancestors = 256538945,256221990)

Reply:
not sure what you mean. This is how other samples look like.

---
In reply to: [256622797](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256622797) [](ancestors = 256622797,256538945,256221990)